source/includes/fact-online-archive-callout.rst

```diff
@@ -0,0 +1,5 @@
+If you are exporting stale data to save on storage costs, consider
```

Comment:
[note] not sure if the word "stale" is beneficial or not. Happy to remove or go with some other wording.

Reply:
I wonder if we should explicitly state "archiving", "If you are archiving stale or old data to..."

Reply:
Good call. Going with "archiving stale data"